src/plone/api/tests/test_content.py

```diff
@@ -376,6 +376,46 @@ def test_create_at_with_title_in_request(self):
 
         self.assertEqual(page.title, 'Test document')
 
+    @unittest.skipIf(HAS_PACONTENTYPES, 'Archetypes only')
+    def test_create_at_event(self):
+        """https://github.com/plone/plone.api/issues/364"""
+        from DateTime import DateTime
+        today = DateTime()
+        tomorrow = today + 1
+        event = api.content.create(
+            container=self.portal,
+            type='Event',
+            title=u'My event',
+            startDate=today,
+            endDate=tomorrow,
+        )
+        self.assertEqual(event.startDate, today)
+        self.assertEqual(event.endDate, tomorrow)
+        results = api.content.find(Title=u'My event')
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].start, today)
+        self.assertEqual(results[0].end, tomorrow)
+
+    @unittest.skipUnless(HAS_PACONTENTYPES, 'Dexterity only')
+    def test_create_dx_event(self):
+        """https://github.com/plone/plone.app.contenttypes/issues/465"""
```

Comment:
@hvelarde  what's the relationship between plone/plone.app.contenttypes#465 and ```plone.api```. It seems to me more a problem of ```plone.app.contenttype```.  What's purpose of this test?

@gbastien FYI.